FreeRTOS/FreeRTOS
# Fleet provisioning winsim demo - fix warnings and reduce demo path length

Description
-----------
This PR fixes build warnings for fleet provisioning demo and reduces the path length of the demo from `.\FreeRTOS-Plus\Demo\AWS\Fleet_Provisioning_Windows_Simulator\Fleet_Provisioning_With_CSR_Demo\` to` .\FreeRTOS-Plus\Demo\AWS\Fleet_Provisioning_Windows_Simulator\CSR_Demo\`  to reduce the possibility of build errors on Windows as Visual Studio enforces a total absolute path length of file to be maximum 260 characters.

Test Steps
-----------

Ran the demo on Visual Studio 2022.

Checklist:
----------



Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
